docs/source/usersguide/random_ray.rst

```diff
@@ -447,6 +448,178 @@ in the `OpenMC Jupyter notebook collection
     separate materials can be defined each with a separate multigroup dataset
     corresponding to a given temperature.
 
+.. _mgxs_gen:
+
+-------------------------------------------
+Generating Multigroup Cross Sections (MGXS)
+-------------------------------------------
```

Comment:
This is great! Consider putting it in it's own section as this is applicable to more than just the random ray solver :)

Reply:
That's a great point. The docs currently only have a very small section on multigroup materials, with most of the info being in a linked jupyter notebook example. It would be much more preferable to have a full top level section in the user guide on how to do MGXS generation in general. However, this particular section within the random ray section seems good to retain for now as it covers how to generate data _specifically_ for use with the random ray solver, rather than all the more general features of MGXS generation in OpenMC. So, for now, I would vote to leave a top level MGXS generation section as a task for a future PR.